geetools/ee_dictionary.py

```diff
@@ -80,3 +82,172 @@ def getMany(self, list: list | ee.List) -> ee.List:
                 d.geetools.getMany(["foo", "bar"]).getInfo()
         """
         return ee.List(list).map(lambda key: self._obj.get(key))
+
+    def toTable(self, valueType: ee.List | ee.Dictionary | Any = Any) -> ee.FeatureCollection:
+        """Convert a :py:class:`ee.Dictionary` to a :py:class:`ee.FeatureCollection` with no geometries (table).
+
+        There are 3 different type of values handled by this method:
+
+        1. Any (default): when values are a `ee.String` or `ee.Number`, the
+        keys will be saved in the column "system:index" and the values in the
+        column "value".
+
+        .. code-block:: python
+
+            import ee, geetools
+
+            ee.Initialize()
+
+            d = ee.Dictionary({"foo": 1, "bar": 2})
+            d.geetools.toTable().getInfo()
+
+            >> {
+              'type': FeatureCollection,
+              'columns': {
+                'system:index': String,
+                'value': String
+              },
+              features: [
+                {
+                  'type': Feature,
+                  'geometry': null,
+                  'id': 'foo',
+                  'properties': {
+                    'value': 1,
+                  },
+                },
+                {
+                  'type': Feature,
+                  'geometry': null,
+                  'id': 'bar',
+                  'properties': {
+                    'value': 2,
+                  },
+                }
+            ]}
```

Comment:
I would move it to an `Examples` section. you can do the 3 of them and jupyter execute will display the results.

Reply:
what about the output? is it ok as I did?

```
>> this is the content of the output
```

Reply:
If you use the Jupyter_execute directive as in the examples of other methods the result is computed on the fly

Reply:
ok, so I could remove the "expected result" 👍